cmd/etcd.go

```diff
@@ -180,6 +187,42 @@ func RestoreEtcdSnapshot(
 	return APIURL, caCrt, clientCert, clientKey, certs, err
 }
 
+func validateCerts(state cluster.State) error {
+	var failedErrs error
+
+	if state.RancherKubernetesEngineConfig == nil {
+		// possibly already started a restore
+		return nil
+	}
+	for name, certPKI := range state.CertificatesBundle {
+		if strings.Contains(name, "token") || strings.Contains(name, "header") || strings.Contains(name, "admin") {
```

Comment:
Can we change this to match the constants in https://github.com/rancher/rke/blob/master/pki/constants.go?